sopel/modules/reddit.py

```diff
@@ -150,13 +150,13 @@ def redditor_info(bot, trigger, match=None):
         is_cakeday = cakeday_start <= now <= cakeday_start + day
 
     if is_cakeday:
-        message = message + ' | 13Cake day'
+        message = message + ' | ' + bold(color('Cake day', colors.LIGHT_PURPLE))
     if commanded:
         message = message + ' | https://reddit.com/u/' + u.name
     if u.is_gold:
-        message = message + ' | 08Gold'
+        message = message + ' | ' + bold(color('Gold', colors.YELLOW))
     if u.is_mod:
-        message = message + ' | 05Mod'
+        message = message + ' | ' + bold(color('Mod', colors.GREEN))
```

Comment:
This is technically _changing_ the color, not just replacing it with formatting functions.

That said… green is the color Reddit uses for displaying moderators' names, so… Consider this a start on the devilish festivities for v6.6.6, since we're changing behavior (however slightly) in a patch release. :stuck_out_tongue:

Reply:
Correct on all counts.
It does change behavior, but if this breaks something... whoever built something that parses sopel's output was doing very silly things anyway, and the changes to spacing and colon bolding would also likely break it.